HEATDamage.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+
+namespace PhysicalDamage.Core
+{
+    class HEATDamage:Damage
+    {
+        public struct HEATData
```

Comment:
Rename to `HeatData`. General rule is that if an acronym is greater than 2 characters in length (example: HTTP) then it should be named with camel case (e.g. HttpExample).